Packs/TrendMicroVisionOne/Integrations/TrendMicroVisionOneV3/TrendMicroVisionOneV3.yml

```
@@ -505,160 +660,200 @@ script:
+    - contextPath: VisionOne.Download_Suspicious_Object_list.risk_level
+      description: Risk level of the analyzed object.
       type: string
-    - contextPath: VisionOne.Download_Suspicious_Object_list.rootSha1
-      description: The rootSha1 value for the object
+    - contextPath: VisionOne.Download_Suspicious_Object_list.root_sha1
+      description: status code for the command.
       type: string
-    - contextPath: VisionOne.Download_Suspicious_Object_list.ip
-      description: The endpoint ip associated with the submission
+    - contextPath: VisionOne.Download_Suspicious_Object_list.analysis_completion_date_time
+      description: The analysis completion date and time.
+      type: string
+    - contextPath: VisionOne.Download_Suspicious_Object_list.expired_date_time
+      description: The expiration date and time for the suspicious object.
       type: string
-    description: Downloads the suspicious object list associated to the specified object. Note ~ Suspicious Object Lists are only available for objects with a high risk level.
-    name: trendmicro-visionone-download-suspicious-object-list
   - arguments:
-    - description: The submission ID for the object submitted to sandbox for analysis
+    - description: The submission ID for the object submitted to sandbox for analysis.
       name: submission_id
       required: true
-    - description: Optional name for the package to be downloaded
-      name: filename
+    - description: If script should wait until the task is finished before returning the result, enabled by default.
+      name: poll
+      auto: PREDEFINED
+      predefined:
+      - "true"
+      - "false"
+      default: true
+    - description: Maximum time to wait for the result to be available.
+      name: poll_time_sec
     outputs:
-    - contextPath: VisionOne.Download_Analysis_Report.submissionId
-      description: The submission for the file
+    - contextPath: VisionOne.Download_Analysis_Report.submission_id
+      description: The submission ID for the sandbox object.
+      type: string
+    - contextPath: VisionOne.Download_Analysis_Report.result_code
+      description: Result code of making a request to download analysis report.
+      type: string
+    - contextPath: VisionOne.Download_Analysis_Report.message
+      description: Message notifying user that analysis report is ready for download.
       type: string
-    - contextPath: VisionOne.Download_Analysis_Report.code
-      description: Response status code for the command
-      type: number
     description: Downloads the analysis report for an object submitted to sandbox for analysis based on the submission ID.
     name: trendmicro-visionone-download-analysis-report
   - arguments:
     - description: URL pointing to the location of the file to be submitted.
       name: file_path
       required: true
-    - description: Name of the file to be analyzed
+    - description: Name of the file to be analyzed.
       name: filename
```

as all other arg names are with `_`
please fix also in the py file 

```suggestion
      name: file_name
```

---

corrected.

---

@shaqnawe 
it's look like in the py file it's still`file_name = args.get(FILE_NAME, EMPTY_STRING)`
which  `FILE_NAME = "filename"`
if this is complicated to be changed in the py file, revert back to use `filename`